src/gui/folder.cpp

```diff
@@ -1027,6 +1027,12 @@ void Folder::startSync()
     }
 
     _engine->setIgnoreHiddenFiles(_definition.ignoreHiddenFiles);
+    if (_allowRemoveAllOnce) {
```

Comment:
What does "once" mean in this context?

Reply:
Only for the next sync

Reply:
So if this is set, the user was prompted already?

Reply:
yes